src/widget/wknobcomposed.cpp

```diff
@@ -38,6 +40,14 @@ void WKnobComposed::setup(QDomNode node, const SkinContext& context) {
     if (context.hasNode(node, "MaxAngle")) {
         m_dMaxAngle = context.selectDouble(node, "MaxAngle");
     }
+
+    if (context.hasNode(node, "RotationOffsetX")) {
```

Comment:
We should keep the neames consistent with your Ring PR. 
There we use currently "RingCenterXOffset", so this schould become "KnobCenterXOffset".